protocol-otr-extension-j.md

```diff
@@ -72,15 +74,15 @@ There are a number of places in OTR version 3 that uses SHA-1 or SHA2-256 in ord
     - The Authenticator should be calculated using SHA3-256(K||M), instead of SHA1-HMAC.
 - Computing AES keys, MAC keys and the secure session ID:
     - Redefine `h1()` to use SHA3-256 instead of SHA1.
-    - Redefine `h2()` to use SHA3-256 instead of SHA256.
+    - Redefine `h2()` to use SHA3-256 instead of SHA-256.
```

Comment:
The rest of the document uses SHA256, not SHA-256 - and the OTR protocol description also uses SHA256.